src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -2662,10 +2662,14 @@ def get_test_privacy_request_results(
     # Escape datetime and ObjectId values
     raw_data = privacy_request.get_raw_access_results()
     escaped_json = json.dumps(raw_data, indent=2, default=storage_json_encoder)
-    escaped_data = json.loads(escaped_json)
+    results = json.loads(escaped_json)
 
     return {
         "privacy_request_id": privacy_request.id,
         "status": privacy_request.status,
-        "results": escaped_data,
+        "results": (
+            results
+            if CONFIG.security.dsr_testing_tools_enabled
+            else "DSR testing tools are not enabled, results will not be shown."
```

Comment:
Hm, I feel like since this whole endpoint deals with the privacy request results, we should just return 403 instead of returning just the id and status of the request. Feels weird to create the results, too, even if the flag is disabled.

Thoughts?

Reply:
I did this as a quick way to hide the results, but not change the way the front-end checks for a completed test run. The front-end checks for the status so if it can't find it, it doesn't know to stop polling. I think a better approach would be for the 403 to stop the polling but I wanted to minimize the changes so close to release. I can do this in a follow up ticket if that's ok?